samples/bot-adaptive-card-actions/csharp/assets/sample.json

```diff
@@ -0,0 +1,68 @@
+[
+  {
+    "name": "officedev-microsoft-teams-samples-bot-adaptivecard-actions-csharp",
+    "source": "officeDev",
+    "title": "Get adaptivecard Actions",
+    "shortDescription": "Demonstrating the feature where user can send adaptivecard actions using bot.",
+    "url": "https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/bot-adaptivecard-actions/csharp",
+    "longDescription": [
+      "Demonstrating the feature where user can send adaptivecard actions using bot."
+    ],
+    "creationDateTime": "2022-12-27",
+    "updateDateTime": "2022-12-27",
+    "products": [
+      "Teams"
+    ],
+    "metadata": [
+      {
+        "key": "TEAMS-SAMPLE-SOURCE",
+        "value": "OfficeDev"
+      },
+      {
+        "key": "TEAMS-SERVER-LANGUAGE",
+        "value": "csharp"
+      },
+      {
+        "key": "TEAMS-SERVER-PLATFORM",
+        "value": "netframework"
+      },
+      {
+        "key": "TEAMS-FEATURES",
+        "value": "bot"
+      }
+    ],
+    "thumbnails": [
+      {
+        "type": "Image",
+        "order": 100,
+        "url": "https://raw.githubusercontent.com/OfficeDev/Microsoft-Teams-Samples/main/samples/bot-suggested-actions/csharp/SuggestedActions/Images/AdaptiveCardActions.gif",
+        "alt": "Solution UX showing suggested action in Bots."
```

Comment:
Solution UX showing Adaptive Card supported actions in bot.

Reply:
fixed